dependencies.yaml

```diff
@@ -22,4 +22,12 @@ repositories:
   eigen_checks:
     type: git
     url: https://github.com/ethz-asl/eigen_checks.git
+    version: master
+  minkindr_ros:
```

Comment:
I think we do not need minkindr... this is supposed to be for 3D transformations, and I do not think we use any now. Could you indicate where is it used? @friedhelm-h  thanks.

Reply:
It was listed in the package.xml and resulted in a compile error this is why I thought it is needed.

Reply:
We still the GSL dependency in the README, correct? @friedhelm-h

Reply:
done! Thanks @friedhelm-h